### ISSUE TITLE
Error Creating clould_controller_clock 

### ISSUE THREAD

**Issue by roshan-** [cloudfoundry-attic/cf-release#1205]
I'm trying to upload a release to AWS, but got this error..

00:02:25 | Extracting release: Extracting release (00:00:00)
00:02:25 | Verifying manifest: Verifying manifest (00:00:00)
00:02:26 | Resolving package dependencies: Resolving package dependencies (00:00:00)
00:02:26 | Processing 66 existing packages: Processing 66 existing packages (00:00:00)
00:02:26 | Creating new jobs: cloud_controller_clock/b0fd1e3b5aefa5d5261404126bc8cdfccd818fa3 (00:00:00)
            L Error: Missing template file 'newrelic.yml.erb' for job 'cloud_controller_clock'

00:02:26 | Error: Missing template file 'newrelic.yml.erb' for job 'cloud_controller_clock'

Started  Sat May 27 00:02:25 UTC 2017
Finished Sat May 27 00:02:26 UTC 2017
Duration 00:00:01

Task 9 error

Uploading release file:
  Expected task '9' to succeed but was state is 'error'

Exit code 1

**Comment by dsabeti** [comment]
Hi @roshan-, what commands did you run to do this? Did you try to create a dev release (with `bosh create release; bosh upload release`, or were you uploading a final release `bosh upload release releases/cf/cf-X.yml`?

**Comment by roshan-** [comment]
I was following steps outlined here...
https://docs.cloudfoundry.org/deploying/aws/index.html

I was successful all the way till the upload-release...
I have run ./scripts/update to ensure my repository is completely up-to-date

$ bosh upload-stemcell ~/downloads/light-bosh-stemcell-3202-aws-xen-hvm-ubuntu-trusty-go_agent.tgz
$ bosh create-release
$ bosh upload-release

**Comment by dsabeti** [comment]
**tl;dr** I'd advise using the other workflow:
```
cd cf-release
bosh upload-release releases/cf/cf-262.yml
```

_More context:_
The v2 BOSH CLI doesn't (yet) support releases that use symlinks ([there's a PR to change this](https://github.com/cloudfoundry/bosh-cli/pull/232)), so if you want to use the "create release" workflow, you actually still have to use the _old_ BOSH CLI:
```
git clone https://github.com/cloudfoundry/cf-release
cd cf-release
./scripts/update

old-bosh create release
old-bosh upload release
bosh deploy ...
```

This workflow is only important for development workflows, though. If you're planning to use a stable version of CF anyway, it's better to just upload the "pre-made" release:
```
git clone https://github.com/cloudfoundry/cf-release
cd cf-release

bosh upload-release releases/cf/cf-262.yml
```

**Comment by roshan-** [comment]
Thanks.. I'll try this.

FWIW, I'm running this in the Windows Linux Subsystem (Ubuntu 16.04). Although, so far, it has behaved exactly like a linux os. So, I'm assuming that wouldn't be an issue

**Comment by roshan-** [comment]
Update: I just did bosh upload-release releases/cf/cf-262.yml

and I got this...

Expected to find 'debian_nfs_server/aac05f22582b2f9faa6840da056084ed15772594'

**Comment by dsabeti** [comment]
Hey @roshan-, I'm not sure what's causing that issue, but if you use the old (ruby) bosh CLI, `bosh upload release` should work.

**Comment by dsabeti** [comment]
Hey @roshan-, any updates on this?

**Comment by magnetnoc** [comment]
I have gotten the exact same error message:

Expected to find 'debian_nfs_server/aac05f22582b2f9faa6840da056084ed15772594'

I tried using boshv1 to fix this, as you suggested, however I then get this error:

$ bosh-1 upload release
RSA 1024 bit CA certificates are loaded due to old openssl compatibility
[WARNING] cannot access director, trying 4 more times...
[WARNING] cannot access director, trying 3 more times...
[WARNING] cannot access director, trying 2 more times...
[WARNING] cannot access director, trying 1 more times...
cannot access director (SSL_connect returned=1 errno=0 state=SSLv2/v3 read server hello A: sslv3 alert handshake failure)

Which is referenced here:

https://github.com/cloudfoundry-incubator/bosh-google-cpi-release/issues/166

And the solution is to use Boshv2

So, not sure how to fix this issue?  Can't use boshv2 because of one error, can't use boshv1 because of another error.  Solutions?

**Comment by dsabeti** [comment]
@magnetnoc Thanks for pointing that out. We're going to try and debug the issue with the BOSH team. In the meantime, if you aren't managing a production environment, I might suggest taking a look at [cf-deployment](https://github.com/cloudfoundry/cf-deployment) -- it works nicely with the bosh2 CLI.

**Comment by magnetnoc** [comment]
Ok, I did try "cf-deployment" and I've run into yet another bug - https://github.com/cloudfoundry/cf-deployment/issues/166

BOSH v1 - bug
BOSH v2 - bug
cf-deployment - bug

not sure how to proceed ...

**Comment by dsabeti** [comment]
Hey @magnetnoc. Sorry about all these bugs! There are two different fixes in the works:

1. **Bosh v2 CLI**: [There's a commit](https://github.com/cloudfoundry/bosh-cli/commit/105157c38524ddbb8367c868afa06eb00f285125) working it's way through the testing pipeline. We're hoping to have the fix release in the next day.
2. **bbl**: We think we've diagnose the issue, and [there's a story](https://github.com/cloudfoundry/bosh-cli/commit/105157c38524ddbb8367c868afa06eb00f285125) in the bbl team's backlog to address the issue, so hopefully there will be a bbl update soon. (I'll update your other issue with details.)

**Comment by parthasarathi204** [comment]
This issue still persists. I have built the latest release and while uploading errors out as     
       L Error: Missing template file 'newrelic.yml.erb' for job 'cloud_controller_clock'

14:25:05 | Error: Missing template file 'newrelic.yml.erb' for job 'cloud_controller_clock'
Usinh bosh cli v2

**Comment by dsabeti** [comment]
@parthasarathi204, as I mentioned in https://github.com/cloudfoundry/cf-release/issues/1224, you'll need to use the Ruby CLI (v1) to run `bosh create release`, and the new CLI to do anything else.

**Comment by davewalter** [comment]
This repository is no longer supported. Closing issue.
